docs/reference/commandline/login.md

```diff
@@ -85,6 +85,8 @@ you can download them from:
 - Microsoft Windows Credential Manager: https://github.com/docker/docker-credential-helpers/releases
 - [pass](https://www.passwordstore.org/): https://github.com/docker/docker-credential-helpers/releases
 
+#### Usage
```

Comment:
Thinking if we should use a more descriptive title; perhaps "Credentials store configuration", or "Configure the credentials store" ?

(open to suggestions 😅)

Reply:
I went for the `Configure the ...` way. Both ways already existed:
```
run.md:787:### Configure namespaced kernel parameters (sysctls) at runtime
pull.md:40:### Proxy configuration
```